openapi/components/requestBodies/PatchTransactionRequest.yaml

```
@@ -1,9 +1,6 @@
-content:
-  application/json:
-    schema:
-      type: object
-      properties:
-        customFields:
-          $ref: ../schemas/ResourceCustomFields.yaml
+content: 
+  application/json: 
+    schema: 
+      $ref: ../schemas/Transactions/PatchTransactionRequest.yaml 
```

Remove trailing spaces
```suggestion
content:
  application/json:
    schema:
      $ref: ../schemas/Transactions/PatchTransactionRequest.yaml
```